Procfile_worker_short

```diff
@@ -0,0 +1 @@
+scheduler: python manage.py rqscheduler
```

Comment:
I'm not clear why the `scheduler` is here in the short queue, instead of in the main Procfile?

Reply:
Once we have a short queue with its own worker, that will run here in the same dyno as the scheduler